.github/workflows/publish.yml

```diff
@@ -0,0 +1,53 @@
+name: Publish
+
+on:
+  release:
+    types: [created]
+
+env:
+  FORCE_COLOR: 1
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+
+      - uses: actions/setup-python@8d9ed9ac5c53483de85588cdf95a591a75ab9f55 # v5.5.0
+        name: Install Python
+        with:
+          python-version: "3.x"
+          allow-prereleases: false
```

Comment:
This step might not even be necessary.

Reply:
Is `pipx` with the latest Python version available by default on these CI runners? Or do you mean specifically `allow-prereleases: false`, which is redundant because `allow-prereleases` already [defaults to `false`](https://github.com/actions/setup-python/blob/main/docs/advanced-usage.md#allow-pre-releases)?